distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -250,6 +249,67 @@ public void testFtpsPut() throws Exception {
     }
 
     /**
+     * Test the ftp command sequence of uploading a dot-file (i.e. .foo) and then renaming that
+     * file to the final filename (i.e. test.txt). Confirm that the catalog contains only one
+     * object and the title of that object is "test.txt".
+     */
+    @Test
+    public void testFtpsPutDotFile() throws Exception {
+
+        setClientAuthConfiguration(NEED);
+        FTPSClient client = createSecureClient(true);
+
+        String dotFilename = ".foo";
+        String finalFilename = "test.txt";
+
+        ftpPut(client, dotFilename, SAMPLE_DATA);
+        boolean ret = client.rename(dotFilename, finalFilename);
+        assertTrue(ret);
+
+        Response response = executeOpenSearch("xml", "q=*", "count=100");
+
+        response.then()
+                .body("metacards.metacard.size()", equalTo(1));
+        response.then()
+                .body("metacards.metacard.string.findAll { it.@name == 'title' }.value",
+                        equalTo(finalFilename));
+
+        // clean up test data
+        TestCatalog.deleteMetacard(getMetacardIdFromResponse(response));
+        disconnectClient(client);
+
+    }
+
+    @Test
+    public void testFtpsMkdirCwdPutFile() throws Exception {
+
+        setClientAuthConfiguration(NEED);
+        FTPSClient client = createSecureClient(true);
+
+        String newDir = "newDirectory";
+        String finalFilename = "test.txt";
+
+        boolean ret = client.makeDirectory(newDir);
+        assertTrue(ret);
+        ret = client.changeWorkingDirectory(newDir);
+        assertTrue(ret);
+        ftpPut(client, finalFilename, SAMPLE_DATA);
+
+        Response response = executeOpenSearch("xml", "q=*", "count=100");
+
+        response.then()
```

Comment:
Same here. We should wrap these in a `WaitCondition`s to avoid a race condition.